src/components/settings-menu/components/MenuContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+import { Box, Stack } from '@rainbow-me/design-system';
+
+interface MenuContainerProps {
+  children: React.ReactNode;
+  testID?: string;
+}
+const MenuContainer = ({ children, testID }: MenuContainerProps) => {
+  return (
+    <ScrollView scrollEventThrottle={32} testID={testID}>
+      <Box
+        paddingBottom={{ custom: 34.5 }}
```

Comment:
34.5 feels very custom, this was on designs?

Reply:
there was definitely a reason for this... let me think lol

Reply:
<img width="220" alt="Screen Shot 2022-08-15 at 4 31 03 PM" src="https://user-images.githubusercontent.com/15272675/184732023-e48047f8-527a-4e49-a1a6-d465822c5d94.png">

i wanted to replicate the implicit top padding here

Reply:
ended up reworking `MenuContainer` a bit in ios to improve the scroll view, so the above response is obsolte